sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdRequestHeaders.java

```diff
@@ -749,6 +752,33 @@ private void addCorrelatedActivityId(final Map<String, String> headers) {
         }
     }
 
+    private void addPriorityLevel(final Map<String, String> headers)
+    {
+        final String value = headers.get(HttpHeaders.PRIORITY_LEVEL);
+
+        final PriorityLevel priorityLevel = EnumUtils.getEnumIgnoreCase(
+            PriorityLevel.class,
+            value);
+
+        if (priorityLevel == null) {
```

Comment:
as discussed offline, priority level header is optional, and it will be default to PriorityLevel.High, so there is no need to throw exception here